### ISSUE TITLE
Criar notificações para mensagens não lidas

### ISSUE THREAD

**Issue by lucasretondo** [PraConstruir/customer_mobile_app#247]
Precisamos criar notificações para alertar o usuário quanto existirem mensagens não lidas.

Podemos colocar um ícone de uma carta no menu ao lado a bolinha laranja:
![image](https://user-images.githubusercontent.com/20990100/33073617-94f363c0-ceaa-11e7-9912-8a96c32d07ad.png)

Nas telas do meu resumo, meu orçamento e minhas negociações ao lado do status:
![image](https://user-images.githubusercontent.com/20990100/33073640-a7443928-ceaa-11e7-8e7f-b99b26567b1f.png)
![image](https://user-images.githubusercontent.com/20990100/33073668-c0cb64fc-ceaa-11e7-89c7-7716ea33f64f.png)
![image](https://user-images.githubusercontent.com/20990100/33073671-c2f36db0-ceaa-11e7-9e38-7922f5cb4606.png)

E nos orçamentos e negociações podemos colocar uma bolinha no menu superior do lado do texto mensagem:
![image](https://user-images.githubusercontent.com/20990100/33073703-e0782f2e-ceaa-11e7-994f-e6c3a08920e8.png)

![image](https://user-images.githubusercontent.com/20990100/33073707-e58186c8-ceaa-11e7-8a1a-ea1488ce6819.png)

**Comment by raphaelMCarvalho** [comment]
@lucasretondo Não existe atualmente um controle de leitura de notificações, esta é uma feature que teremos que desenvolver no backend e nos clientes Android, iOS e web. Vou estimar com o @claudineicaetano.

**Comment by lucasretondo** [comment]
@raphaelMCarvalho sim, não é bug, é novo desenvolvimento. Coloquei aqui para não esquecer.
